src/components/SettingsDialog/SettingsDialog.vue

```diff
@@ -45,13 +45,15 @@
 		<AppSettingsSection v-if="!isGuest"
 			:title="t('spreed', 'Privacy')"
 			class="app-settings-section">
-			<input id="read_status_privacy"
+			<CheckboxRadioSwitch
+				id="read_status_privacy"
 				:checked="readStatusPrivacyIsPublic"
 				:disabled="privacyLoading"
 				type="checkbox"
 				name="read_status_privacy"
 				class="checkbox"
-				@change="toggleReadStatusPrivacy">
+				@update="toggleReadStatusPrivacy"
+			/>
```

Comment:
Expected no line breaks before closing bracket, but 1 line break found

Reply:
Ok, thanks, I fixed it